listenbrainz/db/model/stats/user/artist.py

```diff
@@ -0,0 +1,39 @@
+import pydantic
+
+from datetime import datetime
+from enum import Enum
+from typing import Optional, List
+
+
+class StatisticsRange(Enum):
+    week = 'week'
+    month = 'month'
+    year = 'year'
+    all_time = 'all_time'
+
+
+class UserArtistRecord(pydantic.BaseModel):
+    artist_msid: Optional[str]
+    artist_mbids: List[str] = []
+    listen_count: int
+    artist_name: str
+
+
+class UserArtistStatForRange(pydantic.BaseModel):
+    to_ts: float
+    from_ts: float
+    count: int
+    artists: List[UserArtistRecord]
+
+
+class UserArtistStatJson(pydantic.BaseModel):
+    week: Optional[UserArtistStatForRange]
+    year: Optional[UserArtistStatForRange]
+    month: Optional[UserArtistStatForRange]
+    all_time: Optional[UserArtistStatForRange]
+
+
+class UserArtistStatRow(pydantic.BaseModel):
```

Comment:
terminology: maybe remove `Row`? We know it's a single thing, so maybe it doesn't need it.